src/renderers/dom/shared/HTMLDOMPropertyConfig.js

```diff
@@ -209,6 +229,54 @@ var HTMLDOMPropertyConfig = {
   },
   DOMPropertyNames: {
   },
+  DOMMutationMethods: {
+    value: function(node, next) {
```

Comment:
Ah heck, I'll just bring it over while I'm thinking about it. That way things are at least up to date. Thanks for the prod!